config/routes.rb

```diff
@@ -0,0 +1,19 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :splash, only: [:index]
+
+  resources :groups, only: [:index, :new, :create] do
+    resources :transacs, only: [:index]
```

Comment:
- [OPTIONAL] There should have more than one route (show, create, and new ) in the config/routes.rb file for transacs. Kindly let's add this. ✍🏻